line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/support/LineMessageHandlerSupport.java

```diff
@@ -180,7 +180,8 @@ void dispatch(Event event) {
             dispatchInternal(event);
         } catch (InvocationTargetException e) {
             log.trace("InvocationTargetException occurred.", e);
-            log.error(e.getCause().getMessage(), e.getCause());
+            final Throwable t = e.getCause() == null ? e : e.getCause();
```

Comment:
How about just change loglevel of above log and remove this line? I don't think there is no merit to keep 2 logs here.